### PR TITLE
Update the-changelog-311.md

### PR DESCRIPTION
**Pull request by rob729**

### CODE REVIEW

**Comment by jerodsanto** [comment]
@rob729 what's actually improved here? I'm not seeing it...

**Comment by adamstac** [comment]
Looks like whitespace on the last line is being removed.

**Comment by rob729** [comment]
I have updated the changes 
Please have a look at them

**Comment by adamstac** [comment]
While you're at it, update "Io" to something like "io.js" since that's the true name and would read better.

**Comment by rob729** [comment]
Can you tell me what changes I need to make in my PR ?

File: podcast/the-changelog-300.md [hunk 1]
```diff
@@ -297,10 +297,7 @@ I feel like in a lot of readmes, a lot of open source website, it'll say right t
 **Adam Stacoviak:** We were pretty close to that one though. I think there were a lot of community members knee deep into the Io and Node fork and bringing the communities back together. From my perspective, I think there were plenty of community members that were what I would consider just peer-to-peer developers, not corporations that were -- I guess they were leading Io, but then they were also bringing it back to this recombination of Io/Node.
 
 **Zed Shaw:** Right, after they got Joyent to sort of agree to their things. Because Joyent wasn't really running the project ethically; they were doing exactly what I was saying.
-
-**Jerod Santo:** \[unintelligible 00:43:19.27\]
```
Comment (jerodsanto) [inline podcast/the-changelog-300.md]: Here I said: "They were letting it languish"

File: podcast/the-changelog-300.md [hunk 2]
```diff
@@ -297,10 +297,7 @@ I feel like in a lot of readmes, a lot of open source website, it'll say right t
...
-
-**Zed Shaw:** Yeah, they weren't running it well; they were only doing what they wanted and what benefitted them, but then claiming "Oh, this is a community. We're all friends!" So then Io forks for that very reason.
+Yeah, they weren't running it well; they were only doing what they wanted and what benefitted them, but then claiming "Oh, this is a community. We're all friends!" So then Io forks for that very reason.
```
Comment (jerodsanto) [inline podcast/the-changelog-300.md]: This needs a blank line above it and also needs the `**Zed Shaw:**` heading as it comes after my unintelligible statement.

File: podcast/the-changelog-311.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ What I think is happening today in the cloud is we're kind of reimagining that i
 
 The next layer that we all kind of agreed on was Kubernetes, and the problem Kubernetes was solving is "Alright, I don't have one container, I have many of them. How do I deploy them and control them and scale them and keep them running, and kind of solve all the lifecycle problems that exist when I'm running a bunch of containers?" We've all agreed on that as the kind of de facto technology to solve that problem.
 
-Service mesh I think of as the third layer of this new platform that we're building... And the problem service mesh is trying to solve is "How do I actually control the interaction between different applications that I'm writing?" If containers let me write an application, in real life most interesting things are solved by having lots of pieces of applications talk to each other... So how do I actually see and understand how those different components talk to each other and have some control over that? At its heart, that's what service mesh is trying to do, is give us control over the integration between apps.
```
Comment (jerodsanto) [inline podcast/the-changelog-311.md]: I believe all you've changed here is added an extra space in `integration between apps`. Was this a mistake that you didn't mean to commit? Or maybe you changed something else here that I'm not seeing?

File: podcast/the-changelog-300.md [hunk 4]
```diff
@@ -297,10 +297,7 @@ I feel like in a lot of readmes, a lot of open source website, it'll say right t
 **Adam Stacoviak:** We were pretty close to that one though. I think there were a lot of community members knee deep into the Io and Node fork and bringing the communities back together. From my perspective, I think there were plenty of community members that were what I would consider just peer-to-peer developers, not corporations that were -- I guess they were leading Io, but then they were also bringing it back to this recombination of Io/Node.
```
Comment (jerodsanto) [inline podcast/the-changelog-300.md]: Adam was saying you could add `.js` after `Io` to make it clear that he was referring to `IO.js`, but that's not required by any means to get this merged.

File: podcast/the-changelog-311.md (hunk 3 above)
Comment (rob729) [reply podcast/the-changelog-311.md]: Actually, I have replaced the word 'integration' with 'interaction'.

Comment (jerodsanto) [reply podcast/the-changelog-311.md]: Ahh! How'd I miss that? 😆

**Comment by rob729** [comment]
I have done the requested changes.

**Comment by jerodsanto** [comment]
Thanks for contributing! 💚💚💚

**Pull request merged**